electrumsv/restapi_endpoints.py

```diff
@@ -0,0 +1,59 @@
+"""This is designed with extensibility in mind - see examples/applications/restapi. """
+from typing import Dict, List
+
+from aiohttp import web
+from .logs import logs
+from .app_state import app_state
+from .restapi import good_response, get_network_type
+
+# PATHS
+VERSION = "/v1"
+NETWORK = "/{network}"
+BASE = VERSION + NETWORK
+
+
+# Request variables
+class VARNAMES:
+    pass
+
+
+# Request types
+ARGTYPES: Dict[str, type] = {}
+
+
+def __init__(self):
+    pass
+
+
+class HandlerUtils:
+    pass
+
+    # ---- Utility Functions ----- #
+
+
+class DefaultEndpoints(HandlerUtils):
+
+    routes: List[web.RouteDef] = []
```

Comment:
Perhaps a comment here about how the list is shared between instances.

Reply:
I actually don't want to subclass DefaultEndpoints. I would rather keep the routing tables / endpoints and registration for the dapp independent of one another. I think it simplifies matters. 
- Maybe someone wants a clean slate without conflicting namespaces or unwanted endpoints...